decidim-core/app/views/decidim/participatory_processes/_promoted_process.html.erb

```diff
@@ -18,6 +18,11 @@
           <div class="large-6 large-offset-6 columns">
             <%= link_to participatory_process_path(promoted_process), class: "button expanded button--sc" do %>
               <%= t(".take_part", scope: "layouts") %>
+              <% if promoted_process.active_step %>
```

Comment:
When loading the promoted_processes we should eager load the `active_step` association